### PR TITLE
Adds everything except keyStack.listener tests

### PR DESCRIPTION
**Pull request by stayman**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{presets: ["react"]}
```
Comment (tarikjn) [inline .babelrc]: looks like it should'd be necessary

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/iheartradio/KeyStack/issues"
+  },
+  "homepage": "https://github.com/iheartradio/KeyStack#readme",
+  "peerDependencies": {
+    "react": "^15"
+  },
+  "devDependencies": {
+    "babel-core": "^6.10.4",
```
Comment (tarikjn) [inline package.json]: looks like it should'd be necessary

File: test/mocha.opts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+--compilers js:babel-core/register
```
Comment (tarikjn) [inline test/mocha.opts]: still needed?

File: lib/KeyDown.jsx [hunk 4]
```diff
@@ -27,7 +30,5 @@ var KeyDown = React.createClass({
   }
 })
 
-export default KeyDown
-
 // TODO: - look for shouldComponentUpdate optimizations
 //       - verify support for receiving new props
```
Comment (tarikjn) [inline lib/KeyDown.jsx]: let's rename this file to .js

File: .babelrc (hunk 1 above)
Comment (stayman) [inline .babelrc]: For memorialization, we use babel for JSX in test environments

**Pull request merged**